.codecov.yaml

```
@@ -28,11 +28,11 @@ coverage:
 
   # See http://docs.codecov.io/docs/ignoring-paths
   ignore:
-  - "github/*"
-  - "cmd/*"
+  - "github/**/*"
```

```suggestion
  - ".github/**/*"
```

---

good catch :+1: 
how about:
```suggestion
  - "*github/**/*"
```

---

we could use https://docs.codecov.com/docs/codecov-yaml#validate-your-repository-yaml to validate the config file to see if we need `\.github` or `\\.github`. WDYT?

---

`\.github` is wrong, and it looks like that `\\.github` is wrong as well - the codecov already does some escaping automatically, so we can simply use: 
```suggestion
  - ".github/**/*"
```
but TBH, this doesn't matter since the yaml files shouldn't be calculated to the coverage of go tests anyway.